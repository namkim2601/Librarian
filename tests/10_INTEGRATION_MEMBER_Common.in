CoMmOn nomembers
add member member1
add member member2
add member member3
common nobooks
add collection collection.csv
common 100000 100001 nosuchmember
common 100000 100000 100001
common 100000 100001 100002
rent 100000 111111
rent 100000 111119
rent 100000 111112
relinquish all 100000
rent 100001 111118
rent 100001 111111
rent 100001 111112
relinquish all 100001
rent 100002 111111
rent 100002 111112
rent 100002 111117
relinquish all 100002
common 100000 100001 100002
exit
