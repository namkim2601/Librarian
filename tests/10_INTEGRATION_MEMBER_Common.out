user: No members in system.

user: Success.

user: Success.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: No such member in system.

user: Duplicate members provided.

user: No common books.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)

user: Ending Library process.
